test-suite-java/build.gradle

```diff
@@ -8,8 +8,10 @@ plugins {
 
 repositories {
     mavenCentral()
+    maven { url "https://s01.oss.sonatype.org/content/repositories/snapshots/" }
```

Comment:
Create a base module to reduce duplication https://github.com/micronaut-projects/micronaut-project-template/tree/master/buildSrc/src/main/groovy We could apply it for the `test-suite` projects